cmd/sync-tags/main.go

```diff
@@ -182,11 +184,29 @@ func main() {
 	// create or update tags from kTagCommits as local tags with the given prefix
 	createdTags := []string{}
 	for name, kh := range kTagCommits {
+		// temporarily publish only alpha tags
+		if !strings.Contains(name, "alpha") {
+			continue
+		}
+
 		bName := name
 		if *prefix != "" {
 			bName = *prefix + name[1:] // remove the v
 		}
 
+		// check if name is a semver tag
+		// the semver lib requires the v prefix to be dropped before parsing
+		var skipPublishingSemverTag bool
```

Comment:
considering both publishSemverTags and skipPublishingSemverTag in many places is confusing. suggest something more like:

```go
var (
  semverTag = ""
  publishSemverTag = false
)
// if we are publishing semver tags
if *publishSemverTags {
  // and this is a valid v1... semver tag
  if _, semverErr := semver.Parse(name[1:]); semverErr == nil && strings.HasPrefix(name, "v1.") {
    publishSemverTag = true
    semverTag = "v0." + strings.TrimPrefix(name, "v1.")
  }
}
```

and then only consider the `publishSemverTag` boolean in the following logic

Reply:
Done (and tested).